tests/h/streamer/filter_test.py

```diff
@@ -13,6 +13,55 @@ class FakeSocket:
 
 
 class TestFilterHandler:
+
```

Comment:
Change the order of test to match the code: /id, /group, /user, /uri, /references